optimize/fit_params.py

```diff
@@ -106,8 +108,9 @@ def fit(self, dataloader, epochs=300, save_freq=5, print_freq=1):
 
         # Include initial value in training history (if haven't loaded a checkpoint)
         for param in self.relevant_params_list:
-            if len(self.training_history[param]) == 0:
-                self.training_history[param].append(getattr(self.sim_iter, param).item())
+            self.training_history[param + '_target'].append(getattr(self.sim_iter, param).item())
+            #if len(self.training_history[param]) == 0:
+            #    self.training_history[param].append(getattr(self.sim_iter, param).item())
```

Comment:
I think we still need this condition if we want to be able to load checkpoints, otherwise the initial value will get repeated

Reply:
It should be addressed in the current version